yeonjae/programmers-1/행렬의덧셈.js

```diff
@@ -0,0 +1,12 @@
+function solution(arr1, arr2) {
+  let answer = [];
+
+  for (let i = 0; i < arr1.length; i++) {
+    let result = [];
+    for (let j = 0; j < arr1[i].length; j++) {
+      result.push(arr1[i][j] + arr2[i][j]);
+    }
+    answer.push(result);
+  }
+  return answer;
+}
```

Comment:
```suggestion
function solution(arr1, arr2) {
  const answer = [];

  for (let i = 0; i < arr1.length; i++) {
    const temp = [];
    
    for (let j = 0; j < arr1[i].length; j++) {
      temp.push(arr1[i][j] + arr2[i][j]);
    }
    answer.push(temp);
  }
  return answer;
}
```
- answer, result 둘다 의미가 중복되는 경향이 있어서 result를 temp로 변수명을 변경하였습니다.
- 또한, 배열인데 let이 또 쓰여서 const로 변경하였습니다!! 1순위를 const로 작성 후에 재할당이 필요하면 let으로 변경해주세요

Reply:
오 아래 2중 map 깔끔한거같아요! 피드백 확인했습니다!